test/BeeTokenOffering.test.js

```diff
@@ -33,120 +34,84 @@ contract('BeeTokenOffering constructor', function(accounts) {
   async function balanceOf (user) {
       return (await token.balanceOf(user)).toNumber();
   }
-
+/*
   it("should not allow to contribute more than allowed by the cap", async function() {
       await token.setTokenOffering(offering.address, 0);
+      await offering.startOffering(300);
       await whitelistTierA(user3);
       if ((await offering.currentTime()) <= (await offering.doubleTime())) {
         await util.expectThrow(sendTransaction(16, user3));
       }
   });
-
+*/
   it("should sell tokens at a prespecified rate", async function() {
-      await token.setTokenOffering(offering.address, 0);
+      //await token.setTokenOffering(offering.address, 0);
+      //await offering.startOffering(300);
       await whitelistTierA(user2);
 
       // 1 ETH is well below the cap
       const contribution1 = 1;
       await sendTransaction(contribution1, user2);
-      assert.equal(await balanceOf(user2), util.toBee(await offering.rate()));
-      assert.equal((await offering.weiRaised()).toNumber(), util.toEther(contribution1));
+      //assert.equal(await balanceOf(user2), util.toEther(await offering.rate()));
```

Comment:
why not checking balance here ?

Reply:
We're allocating tokens after the end of sale, so their token balance won't show on this section anymore